pkg/telemetry/manager.go

```diff
@@ -77,18 +83,28 @@ type Manager interface {
 	AddConsumer(c Consumer) error
 	// AddWorkflow adds a workflow with providers which will provide telemetry data.
 	AddWorkflow(Workflow)
-	// Execute executes all workflows and returns an aggregated report from those
+	// TODO
```

Comment:
Another TODO we should resolve or have an issue linked for.

Reply:
Leftovers, removed that.